config/metrics_endpoints.json

```diff
@@ -1,635 +1,1111 @@
+            "conwetlab/biz-ecosystem-charging-backend",
+            "conwetlab/biz-ecosystem-logic-proxy",
+            "conwetlab/biz-ecosystem-rss"
+        ]
+    },
+    {
+        "enabler": "Cepheus",
+        "catalogue": "iot-data-edge-consolidation-ge-cepheus",
+        "academy": "",
+        "readthedocs": "fiware-cepheus",
+        "helpdesk": "Cepheus",
+        "backlog": "",
+        "jira_workitem": "",
+        "coverall": "",
+        "coverll": "Orange-OpenSource/fiware-cepheus",
+        "github": [
+            "Orange-OpenSource/fiware-cepheus"
+        ],
+        "docker": [
+            "orangeopensource/fiware-cepheus-cep",
+            "orangeopensource/fiware-cepheus-broker"
+        ]
+    },
+    {
+        "enabler": "Cloud Portal",
+        "catalogue": "self-service-interfaces-cloud-portal",
+        "academy": "Cloud Portal",
+        "readthedocs": "fiware-cloud-portal",
+        "helpdesk": "Cloud Portal",
+        "backlog": "",
+        "jira_workitem": "",
+        "coverall": "",
+        "github": [
+            "ging/fiware-cloud-portal"
+        ],
+        "docker": [
+            "fiware/cloud-portal"
+        ]
+    },
+    {
+        "enabler": "Cloud Rendering",
+        "catalogue": "cloud-rendering",
+        "academy": "Cloud Rendering",
+        "readthedocs": "cloud-renderer",
+        "helpdesk": "Cloud Render",
+        "backlog": "",
+        "jira_workitem": "",
+        "coverall": "",
+        "github": [
+            "tospie/fiware-cloud-rendering-service"
+        ],
+        "docker": []
+    },
+    {
+        "enabler": "Comet - Short Time Historic",
+        "catalogue": "sth-comet",
+        "academy": "",
+        "readthedocs": "fiware-sth-comet",
+        "helpdesk": "STH-Comet",
+        "backlog": "",
+        "jira_workitem": "",
+        "coverall": "",
+        "github": [
+            "telefonicaid/fiware-sth-comet"
+        ],
+        "docker": [
+            "fiware/sth-comet"
+        ]
+    },
+    {
+        "enabler": "Cosmos",
+        "catalogue": "",
+        "academy": "",
+        "readthedocs": "fiware-cosmos-flink",
+        "helpdesk": "Cosmos",
+        "backlog": "",
+        "jira_workitem": "",
+        "coverall": "",
+        "github": [
+            "ging/fiware-cosmos-orion-flink-connector",
+            "ging/fiware-cosmos-orion-flink-connector-examples",
+            "ging/fiware-cosmos-orion-spark-connector",
+            "ging/fiware-cosmos-orion-spark-connector-examples"
+        ],
+        "docker": []
+    },
+    {
+        "enabler": "Cosmos - BigData Analysis",
+        "catalogue": "bigdata-analysis-cosmos",
+        "academy": "Cosmos",
+        "readthedocs": "fiware-cosmos",
+        "helpdesk": "Cosmos",
+        "backlog": "",
+        "jira_workitem": "",
+        "coverall": "",
+        "github": [
+            "telefonicaid/fiware-cosmos",
+            "telefonicaid/fiware-sinfonier"
```

Comment:
delete "telefonicaid/fiware-sinfonier"

Reply:
Fixed in https://github.com/flopezag/fiware-tsc-dashboard/pull/38/commits/55ea9ac2cb5f6bb623a5b2897a9d1e5dede5c585